canonical-web-and-design/releases.demo.haus
# Commit data for releases table

## Done

Added commit data for releases table
Refactor  back-end and front-end to merge into single source of data (all prod, staging domains and repos info are in one place)
Add test for domain-info endpoint
Removed some sites that are down (no jenkins job), archived (github repo) or obsolete (don't exist anywhere).
Adjusted table styling
Removed sites that will not be supported because they have been moved to ubuntu.com (e.g. partners.ubuntu.com, tutorials.ubuntu.com)

## QA

- Checkout this branch `improvements`
- `dotrun -e RELEASES_GITHUB_TOKEN={TOKEN} serve` (ask me for the token)
- Check homepage has the new improved features (more commit info, better dates)

## Issues
Fixes #45
